maps/mjolnir/bos/structures/statue.dm

```diff
@@ -1,41 +1,41 @@
-	icon_state = "shahid"
+    name = "Roger Blackmore statue"
+    desc = "A native of Tersten, a war hero who was head of the Tersten cell of the League of Struggle. Led the takeover of Tersten City, marched in the forefront of the rebels, blew himself up next to one of the army's military installations, posthumously a hero of the Sovereign Lordania."
+    icon_state = "shahid"
 
 /obj/structure/memstatue/matt
-	name = "Matt Striker statue"
-	desc = "Personally killed over ninety SCG commanders during the conflict. Passed away after numerous attempts on his life due to stress."
-	icon_state = "suecidnik"
-
+    name = "Matt Striker statue"
+    desc = "Personally killed over ninety SCG commanders during the conflict. Passed away after numerous attempts on his life due to stress."
+    icon_state = "suecidnik"
+/*
 /obj/structure/memstatue/boris
-	name = "Boris Booker statue"
-	desc = "Your leader and founder of the Sovereign Lordania. 'I see no point in Lordania's existence if it is not united into one strong state capable of dealing with any aggressor in the galactic arena.'"
-	icon_state = "booker"
-
+    name = "Boris Booker statue"
+    desc = "Your leader and founder of the Sovereign Lordania. 'I see no point in Lordania's existence if it is not united into one strong state capable of dealing with any aggressor in the galactic arena.'"
+    icon_state = "booker"
+*/
 /obj/structure/memstatue/vasiliy
-	name = "Vasiliy Petrenko statue"
-	desc = "Effectively used the subjugation implants to sacrifice himself and his personnel to destroy the SCG patrol fleet in the name of sovereign Lordania. The Systems will not forget their heroes!"
-	icon_state = "urod"
+    name = "Vasiliy Petrenko statue"
+    desc = "Effectively used the subjugation implants to sacrifice himself and his personnel to destroy the SCG patrol fleet in the name of sovereign Lordania. The Systems will not forget their heroes!"
+    icon_state = "urod"
 
 /obj/structure/memstatue/grisha
-	name = "Bolshoy Grigoriy statue"
-	desc = " The Hero cat. He bit the wire of an enemy artillery mount and saved the squad."
-	icon_state = "grisha"
+    name = "Bolshoy Grigoriy statue"
+    desc = " The Hero cat. He bit the wire of an enemy artillery mount and saved the squad."
+    icon_state = "grisha"
 
 /obj/structure/memstatue/shmesser
-	name = "Unknown warrior statue"
-	desc = " This naked unknown warrior appeared out of nowhere and attacked Lordania's enemies with only a toolbox. He showed the most important trait of the Lordanian people - courage and commitment to the homeland"
-	icon_state = "shmesser"
+    name = "Unknown warrior statue"
+    desc = " This naked unknown warrior appeared out of nowhere and attacked Lordania's enemies with only a toolbox. He showed the most important trait of the Lordanian people - courage and commitment to the homeland"
```

Comment:
`" This naked unknown` - > `"This unknown, naked warrior` since 'unknown' tends to be before everything else to add some drama and stuff. Also space for some reason.